src/ucp/dt/dt.h

```diff
@@ -57,6 +60,14 @@ typedef struct {
 } ucp_memory_info_t;
 
 
+typedef ucs_status_t (*ucp_dt_pack_func_t)(ucp_worker_h, void*, const void*,
```

Comment:
These 2 types could be united, I preferred to separate them, as they are semantically different and they are currently equal only by coincidence (might change in the future).